### PR TITLE
Add component ID and logo for Python Bottle plugin

### PR DESCRIPTION
**Pull request by jiang1997**
Add Bottle logo

- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why is `skywalking-ui` submodule updated?

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
 * Add component ID(128) for Java Hutool plugin.
 * Add Zipkin query exception handler, response error message for illegal arguments.
 * Fix a NullPointerException in the endpoint analysis, which would cause missing MQ-related `LocalSpan` in the trace.
+* Add component ID(7014) for Python Bottle plugin
```
Comment (kezhenxu94) [inline docs/en/changes/changes.md:18]:
```suggestion
* Add component ID(7015) for Python Bottle plugin
```

**Comment by wu-sheng** [comment]
> > Why is `skywalking-ui` submodule updated?
> 
> I thought the image files were going there. Isn’t it?

What image? I don't see any logo relative PR in that repository.

**Comment by jiang1997** [comment]
> > > Why is `skywalking-ui` submodule updated?
> > 
> > 
> > I thought the image files were going there. Isn’t it?
> 
> What image? I don't see any logo relative PR in that repository.

The plugin logo.
Yes, I did not create any PR for that repo.
I just refer to this https://stackoverflow.com/a/47971286/9845190 to create one pull request across submodules.

**Comment by wu-sheng** [comment]
Please keep the submodule unchanged. No matter if this repo includes the latest commit ID of the UI repo, you don't change it, so, it is irrelevant.

**Comment by jiang1997** [comment]
> Please keep the submodule unchanged. No matter if this repo includes the latest commit ID of the UI repo, you don't change it, so, it is irrelevant.

Right.

**Pull request closed without merging**